website/routes.py

```diff
@@ -175,14 +175,19 @@ def make_url_map(app):
 
     process_rules(app, [
 
-        Rule('/dashboard/', 'get', website_views.dashboard, OsfWebRenderer('dashboard.mako')),
+        Rule('/dashboard/', 'get', website_views.redirect_to_myprojects, OsfWebRenderer('dashboard.mako')),
```

Comment:
The redirect and the `/myprojects/` route are unnecessary. `/dashboard/` should go directly to the dashboard.